spec/commands/exec_spec.rb

```diff
@@ -55,6 +55,13 @@
     expect(out).to eq("hi")
   end
 
+  it "respects custom process title when loading through ruby" do
+    create_file "Gemfile"
+    create_file "a.rb", 'Process.setproctitle("1-2-3-4-5-6-7-8-9-10-11-12-13-14-15-16"); puts `ps -eo args | grep [1]-2-3-4-5-6-7-8-9-10-11-12-13-14-15-16`'
```

Comment:
I can't think of another way to do it, either. It may be worth adding a brief comment explaining how it's testing it, for folks less familiar with *nix tools.

Reply:
Thank you, I'll add a little comment. :+1:

Reply:
I added a local variable with a long name, to explain what the test script is about :smiley: